From b82780cbe4bf3a094ad98d86b0f5165df1b5c898 Mon Sep 17 00:00:00 2001
From: Surface Duo Builder <bariccatti@gmail.com>
Date: Sun, 19 Apr 2026 19:05:42 +0000
Subject: [PATCH] Remove Samsung radio HIDL Java dependencies

vendor/interfaces no longer ships Samsung HIDL stubs (stripped for
Surface Duo). These Samsung radio deps in frameworks/base Android.bp
cause meta_lic build failures.
---
 Android.bp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Android.bp b/Android.bp
index 4ba0ac47d..da4b8fca8 100644
--- a/Android.bp
+++ b/Android.bp
@@ -228,7 +228,6 @@ java_library {
         "android.hardware.gnss-V2.1-java",
         "android.hardware.health-V1.0-java-constants",
         "android.hardware.radio-V1.6-java",
-        "vendor.samsung.hardware.radio-V2.2-java",
         "android.hardware.radio.data-V5-java",
         "android.hardware.radio.ims-V4-java",
         "android.hardware.radio.messaging-V5-java",
@@ -236,7 +235,6 @@ java_library {
         "android.hardware.radio.network-V5-java",
         "android.hardware.radio.sim-V5-java",
         "android.hardware.radio.voice-V5-java",
-        "vendor.samsung.hardware.radio.network-V1-java",
         "android.hardware.thermal-V1.0-java-constants",
         "android.hardware.thermal-V1.0-java",
         "android.hardware.thermal-V1.1-java",
-- 
2.34.1

